bivt-cap/bivt-frontend
# Plugins/expenses/issue26

### Expense plugin (full stack) added:
#Please test on Android environment first 
Test Steps:

#Add a bill
Add a bill by choosing different category
Change the date to previous week or previous month
Check if the tabs are filtering bills correctly
Delete a bill
Check if the total amounts are correct or not.

#Add a budget
Try setting up budgets in different intervals.
Adding or removing the bill should sync with the budget amount of that interval

#Known issues:
Responsiveness of the plugin modal 
Console logs

The most important thing to test is if the amounts are syncing properly or not! We will be creating graphs based on these final amounts.

PS: There are a lot of things going on between bills and budget and it may happen that I might have missed some edge cases. Do let me know if you found such cases.
